src/monitoring_tests/template_test.py

```diff
@@ -130,6 +157,28 @@ def get_decoded_settlement(cls, tx_hash: str):
             decoded_settlement.tokens,
         )
 
+    @classmethod
+    def get_endpoint_order_data(cls, tx_hash: str) -> List[Any]:
+        """
+        Get all orders in a transaction from the transaction hash.
+        """
+        prod_endpoint_url = (
```

Comment:
Could also be barn, right? From the tx hash, we cannot distinguish whether it's barn or prod, so I believe we should also check the barn orderbook here.

Reply:
Just added a lookup at the barn orderbook if the prod one fails

Reply:
Sounds good!